source/plugin/text/representations/configurate.rst

```diff
@@ -0,0 +1,47 @@
+====================
+Configuration Format
+====================
+
+SpongeAPI offers support to serialize text directly to a Configurate configuration file through the use of TypeTokens.
+Text are saved using the same node structure as the Text's JSON representation.
+
+.. tip::
+
+    For information on how to use Configurate to create configuration files for your plugin, please refer to
+    :doc:`/plugin/configuration/index`.
```

Comment:
Ehh, yeah sorry. Didn't notice the second commit, and the preview wasn't updated.